packages/slate-react/src/components/leaf.js

```diff
@@ -5,6 +5,10 @@ import SlateTypes from 'slate-prop-types'
 
 import OffsetKey from '../utils/offset-key'
 
+const attributes = {
+  'data-text': true,
```

Comment:
Can you rename this to `data-slate-leaf: true` to match the other data attributes we've got?

Reply:
Fixed~